src/js/constants/DefaultErrorMessages.js

```diff
@@ -0,0 +1,164 @@
+const DefaultErrorMessages = [
+
+  //
+  // The following rule will match every message path, behaving
+  // like the default sink if a path is not found.
+  //
+  // Remember to concat these error messages when you are composing
```

Comment:
Been there, done that, and it was considered bad idea to write more complicated high-level functions for such a simple task :|